.rspec

```diff
@@ -0,0 +1,2 @@
+--format documentation
```

Comment:
Tohle je spis osobni preference nez nastaveni pro vsechny. Dal bych to pryc. Pokud preferujes sam tenhle format, dej si to radeji do `~/.rspec`